### ISSUE TITLE
lasfilter drops CRS definition

### ISSUE THREAD

**Issue by bw4sz** [Jean-Romain/lidR#126]
Is it possible to retain the projection information after filtering? Am I using the function incorrectly?

```
library(lidR)
LASfile <- system.file("extdata", "MixedConifer.laz", package="lidR")
tile = readLAS(LASfile, select = "xyz", filter = "-drop_z_below 0")
tile@crs<-CRS("+init=epsg:32617")
tile<-tile %>% lasfilter("Z" > 2)
```

```
> tile
class        : LAS (LASF v1.2)
memory       : 1.1 Mb 
extent       : 481253.3 , 481354.9 , 3812921 , 3813021 (xmin, xmax, ymin, ymax)
area         : 10110.4 m² (convex hull)
points       : 47602 points,  NA pulses
density      : 4.71 points/m²,  NA pulses/m²
field names  : X Y Z 
coord. ref.  : NA 
```

Using lidR_1.4.1 

Thanks for this great package.

**Comment by Jean-Romain** [comment]
Hi, thanks for reporting. `lasfilter` is an old function (maybe the oldest) and the CRS support is very young. I miss this point. My bad. I will fix that within an hour.

**Comment by Jean-Romain** [comment]
This is fixed in v1.4.2. I will also fix that in the development branch

```r
devtools::install_github("Jean-Romain/lidR")
```

**Comment by bw4sz** [comment]
Confirmed fix. Thank you.
